src/functionalTest/resources/scenarios/RIA-3651-decide-ftpa-application-appellant-reheard32-case-flag.json

```diff
@@ -0,0 +1,92 @@
+{
+  "description": "RIA-3651 Decide FTPA application appellant FTPA decision (Reheard Rule 32) - feature flag on",
+  "launchDarklyKey": "dlrm-setaside-feature-flag:true",
```

Comment:
I think that you have remove some old feature flags which should be set to true for instance this one. which had the feature flag "reheard-feature:true" previously, it happened with other cases as well, please review them.